Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.yml

```diff
@@ -782,6 +812,9 @@ script:
     - description: PAPI (Ion Standard) property version.
       name: property_version
       type: textArea
+    - description: activation note
+      name: note
+      required: true
```

Comment:
```suggestion
```
The same also this argument cannot be required
Because it breaks backward compatibility